tls/s2n_connection.c

```diff
@@ -1124,3 +1114,14 @@ const uint8_t *s2n_connection_get_sct_list(struct s2n_connection *conn, uint32_t
     *length = conn->ct_response.size;
     return conn->ct_response.data;
 }
+
+int s2n_connection_is_client_auth_enabled(const struct s2n_connection *s2n_connection)
```

Comment:
For consistency update `s2n_allowed_to_cache_connection` call to use this function?

Reply:
Done